core/camel-support/src/main/java/org/apache/camel/support/builder/ExpressionBuilder.java

```diff
@@ -2036,4 +2038,42 @@ public String toString() {
         };
     }
 
+    /**
+     * Returns the expression for the message body as pretty formatted string
+     */
+    public static Expression prettyBodyExpression() {
+        return new ExpressionAdapter() {
+            @Override
+            public Object evaluate(Exchange exchange) {
+                String body = exchange.getIn().getBody(String.class);
+
+                if (body == null) {
+                    return null;
+                } else if (body.startsWith("{") && body.endsWith("}") || body.startsWith("[") && body.endsWith("]")) {
+                    return Jsoner.prettyPrint(body); //json
+                } else if(body.startsWith("<") && body.endsWith(">")) {
```

Comment:
fix checkstyle on code,

run install -P format,fastinstall

Reply:
I ran it and nothing happens. Running with `-P sourcecheck` I could see that checkstyle error is at `core/camel-support/src/test/java/org/apache/camel/support/RestConsumerContextPathMatcherTest.java` which was not changed by this PR.

According with checkstyle this `https://github.com/apache/camel/blob/main/core/camel-support/src/test/java/org/apache/camel/support/RestConsumerContextPathMatcherTest.java#L29` must be `final`

Doing so introduces a new error as below:
```
[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.048 s <<< FAILURE! -- in org.apache.camel.support.RestConsumerContextPathMatcherTest
[ERROR] org.apache.camel.support.RestConsumerContextPathMatcherTest.testRestConsumerContextPathMatcherWithAmbiguousPaths -- Time elapsed: 0.022 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.IllegalStateException to be thrown, but nothing was thrown.
```